consensus/parlia/snapshot.go

```diff
@@ -207,6 +207,17 @@ func (s *Snapshot) updateAttestation(header *types.Header, chainConfig *params.C
 	}
 }
 
+func (s *Snapshot) SignRecently(validator common.Address) bool {
+	for seen, recent := range s.Recents {
+		if recent == validator {
+			if limit := uint64(len(s.Validators)/2 + 1); s.Number+1 < limit || seen > s.Number+1-limit {
```

Comment:
`seen>s.Number+1-limit` includes  the `s.Number+1<limit`, 
remove `s.Number+1<limit` would be fine.

Reply:
I think ‘s.Number+1<limit’ should be kept.
becase  `s.Number+1-limit` would overflow  when s.Number+1 < limit

Reply:
> `seen>s.Number+1-limit` includes the `s.Number+1<limit`, remove `s.Number+1<limit` would be fine.

I think ‘s.Number+1<limit’ should be kept.
becase s.Number+1-limit would overflow when s.Number+1 < limit

Reply:
that's right.